src/atsu/common/src/types/Clock.ts

```diff
@@ -0,0 +1,25 @@
+export class Clock {
+    public year: number;
+
+    public month: number;
+
+    public dayOfMonth: number;
+
+    public hour: number;
+
+    public minute: number;
+
+    public second: number;
+
+    public secondsOfDay: number;
+
+    constructor() {
```

Comment:
You can make a shorthand for this by declaring properties in the constructor:

```ts
constructor(
    public year: number,
    public month: number,
    public dayOfMonth: number,
    public hour: number,
    public minute: number,
    public second: number,
    public secondsOfDay: number,
) {
}
```

TypeScript will automatically generate the assignments in the constructor body at compile-time. The properties in the class body can then be omitted.